src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java

```diff
@@ -0,0 +1,80 @@
+package club.sk1er.patcher.mixins.features;
+
+import club.sk1er.patcher.config.PatcherConfig;
+import gg.essential.universal.UResolution;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiIngame;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraftforge.client.GuiIngameForge;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(GuiIngameForge.class)
+public class GuiIngameForgeMixin_TitleRendering extends GuiIngame {
+
+    @Shadow
+    private FontRenderer fontrenderer;
+
+    public GuiIngameForgeMixin_TitleRendering(Minecraft mc) {
+        super(mc);
+    }
+
+    @Inject(method = "renderTitle", at = @At("HEAD"), cancellable = true, remap = false)
+    private void patcher$cancelTitleRender(CallbackInfo ci) {
+        if (PatcherConfig.disableTitles) ci.cancel();
+    }
+
+    @Inject(method = "renderTitle", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/GlStateManager;scale(FFF)V", ordinal = 0, shift = At.Shift.AFTER))
+    private void patcher$modifyTitle(int l, int age, float opacity, CallbackInfo ci) {
+        GlStateManager.popMatrix();
+        GlStateManager.pushMatrix();
```

Comment:
It appears you're cancelling the scale by `4, 4, 4`, then scaling by `4 * scale, 4 * scale, 4 * scale`? What's the point in resetting the matrix when you could just scale by only the modifier and not care about the `4f`? It'll just nuke people's changes to the matrix for no reason. Same goes for modifying the subtitle. Just scale by the modifier before the `drawString` calls and don't nuke the matrix.